db/schema.rb

```diff
@@ -43,9 +43,11 @@
     t.string   "provider"
     t.string   "uid"
     t.string   "name"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at",               null: false
+    t.datetime "updated_at",               null: false
```

Comment:
It's triyng to get all the null: false / true on one line and `t.string   "encrypted_twitter_secret"` is a bit longer than `t.datetime "created_at"`